api/api_private_account.go

```diff
@@ -334,6 +357,38 @@ func (s *PrivateAccountAPI) SignTransaction(ctx context.Context, args SendTxArgs
 	return &SignTransactionResult{data, signedTx}, nil
 }
 
+// SignTransactionAsFeePayer will create a transaction from the given arguments and
+// tries to sign it as a fee payer with the key associated with args.From. If the given
+// passwd isn't able to decrypt the key, it fails. The transaction is returned in RLP-form,
+// not broadcast to other nodes
+func (s *PrivateAccountAPI) SignTransactionAsFeePayer(ctx context.Context, args SendTxArgs, passwd string) (*SignTransactionResult, error) {
+	// No need to obtain the noncelock mutex, since we won't be sending this
```

Comment:
I think we don't need to say about fee payer's nonce lock.

Reply:
I changed the comment. Please check it again.